app/components/dashboard/diabetes/coverage_card_component.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="d-lg-flex w-lg-50 pr-lg-2">
```

Comment:
[nit] There's a couple of small differences in the LTFU graph:
- Background color of the graph is grey
- Height of the cards has increased
- Description in the LTFU graph says X patients with no visit `since 1-Jun-2022 to 31-Aug-2022`

**Before**
<img width="1387" alt="image" src="https://user-images.githubusercontent.com/16774200/183075570-e1b8f5b1-4251-4a5d-8398-36f4b1cf33e2.png">

**After**
<img width="1397" alt="image" src="https://user-images.githubusercontent.com/16774200/183075587-93d30b55-6f97-4564-aeb0-127d3b5cb43b.png">

Reply:
Done ✅